src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,46 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CalculatorTest {
+
+    @Test
+    void null_또는_빈문자열() {
+        int result = Calculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = Calculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
+
+    @Test
+    void 숫자하나() {
+        assertThat(Calculator.splitAndSum("1")).isEqualTo(1);
+    }
+
+    @Test
+    void 숫자_쉼표구분자() {
+        assertThat(Calculator.splitAndSum("1,2")).isEqualTo(3);
+        assertThat(Calculator.splitAndSum("1,2,3")).isEqualTo(6);
+    }
+
+    @Test
+    void 숫자_쉼표_콜론구분자() {
+        assertThat(Calculator.splitAndSum("1,2:3")).isEqualTo(6);
```

Comment:
```suggestion
        actual = Calculator.splitAndSum("1,2:3");
        
        assertThat(actual).isEqualTo(6);
```

검증단에서는 코드를 넣기보다는
직관적으로 표현하는것을 개인적으로는 지향하고 있어요 :)